embedding/embedlite/embedshared/EmbedLiteWindowChild.cpp

```diff
@@ -199,9 +196,21 @@ void EmbedLiteWindowChild::CreateWidget()
               );
   GetWidget()->UpdateBounds(true);
 
+  // Initialize ScreenManager
+  RefreshScreen(mBounds.Width(), mBounds.Height());
+
   mInitialized = true;
   Unused << SendInitialized();
 }
 
+void EmbedLiteWindowChild::RefreshScreen(int width, int height)
+{
+  LayoutDeviceIntRect rect(0, 0, width, height);
+  AutoTArray<RefPtr<Screen>, 1> screenList;
+  RefPtr<Screen> screen = new Screen(rect, rect, 0, 0, DesktopToLayoutDeviceScale(), CSSToLayoutDeviceScale(), 96);
```

Comment:
These initial values feel wrong. I think we should pass down to the EmbedLiteWindowChild (from QtMozEmbed via EmbedLiteWindowParent) at least dpi and screen color depth. ScreenHelperAndroid::AddScreen uses 24bits for pixel and color depth and has fixed 160.0 for dpi.

Reply:
Surely we can start with this one create more tasks on top of this.